test/screenshot/fab/index.tsx

```diff
@@ -0,0 +1,250 @@
+import * as React from 'react';
+// @ts-ignore
```

Comment:
I'm not sure why the Fab import has to specify .tsx (which is the error being suppressed here). Can you document why this is required in a comment?

Reply:
This is because we still have the index.js files. This is temporary until we get all TSX files in.